assignments/stretch-function-conversion.js

```diff
@@ -2,22 +2,39 @@
 
 // let myFunction = function () {};
 
+let myFunction = () => {};
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 
+let anotherFunction = (param) => param;
```

Comment:
You're able to drop the parentheses if you have exactly one parameter (if you aren't doing anything fancy with it):
```javascript
let anotherFunction = param => param;
```